### ISSUE TITLE
Use context manager

### ISSUE THREAD

**Issue by Stimaleger** [square/pylink#55]
Hey !

Is it in the pipe to add context manager ?

**Comment by hkpeprah** [comment]
Could you elaborate on what you mean?  Like:
```
with pylink.JLink()as jl:
    jl.connect()
```
or?

**Comment by Stimaleger** [comment]
yeah something like that ! 

It's a good solution to add context manager when you use external ressources.

This avoid using (try, except, finally) :
``` python
    jlink = JLink()
    try:
        jlink.open()
        jlink.set_tif(enums.JLinkInterfaces.SWD)
        jlink.connect(chip_name='NRF52832_XXAA'
    finally:
        jlink.close()

```

**Comment by hkpeprah** [comment]
Should be available in the latest release, `v0.2.0`.  Thanks for the work here!
